lib/statix.ex

```diff
@@ -44,24 +44,28 @@ defmodule Statix do
       for pipelining and easily wrapping existing code.
       """
       # TODO: Use `:erlang.monotonic_time/1` when we depend on Elixir ~> 1.2
-      def measure(key, fun) when is_function(fun, 0) do
+      def measure(key, fun, options \\ []) when is_function(fun, 0) do
```

Comment:
It will more convenient if we make `options` second argument in `measure/3` function.